src/operator/cudnn_rnn-inl.h

```diff
@@ -526,7 +526,7 @@ class CuDNNRNNOp : public Operator {
   cudnnRNNInputMode_t input_mode_;
   cudnnDropoutDescriptor_t dropout_desc_;
   Storage::Handle dropout_states_, reserve_space_;
-  uint64_t seed_ = 1337ull;
+  uint64_t seed_ = 17 + rand() % 4096;  // NOLINT(runtime/threadsafe_fn)
```

Comment:
I'll merge this in first.

Later we might consider a static atomic seed that gets incremented each time to be deterministic.